extras/dump_iphone_video.py

```diff
@@ -171,6 +166,13 @@ def main():
     logger = make_db_logger()
     iphone.DISABLE_LSL = True
 
+    # Check if we should be running the dump on this machine.
+    server_name = cfg.get_server_name_from_env()
+    if get_device_assignment('IPhone_dev_1') != server_name:
```

Comment:
This seems like adding noise to the log if two of the three servers will always be logging this (expected) case.